src/test/kotlin/uk/gov/gdx/datashare/helpers/builders/SupplierSubscriptionBuilder.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.gdx.datashare.uk.gov.gdx.datashare.helpers.builders
+
+import uk.gov.gdx.datashare.enums.EventType
+import uk.gov.gdx.datashare.repositories.SupplierSubscription
+import java.time.LocalDateTime
+import java.util.*
+
+data class SupplierSubscriptionBuilder(
+  var supplierSubscriptionId: UUID = UUID.randomUUID(),
+  var supplierId: UUID = UUID.randomUUID(),
+  var clientId: String = "test-client-id",
+  var eventType: EventType = EventType.DEATH_NOTIFICATION,
+  var whenCreated: LocalDateTime = LocalDateTime.now(),
+  var whenDeleted: LocalDateTime? = null,
+  var new: Boolean = true,
```

Comment:
we should probably follow a convention here, either all true or all false, I don't mind which way around it is :)

Reply:
Unfortunately it seems there isn't already a convention - we already have some with true, and some with false

Reply:
eg AcquirerEventBuilder has false AcquirerSubscriptionBuilder is true - I tried to follow the conventions of the matching Acquirer builder for each supplier builder

Reply:
could you make them all match one convention then pls, and blame ethan or I for this extra work 😅😅

Reply:
hahaha it's ethan, you're safe